sdk/storage/Azure.Storage.Files.DataLake/src/DataLakeUriBuilder.cs

```diff
@@ -189,8 +189,8 @@ public DataLakeUriBuilder(Uri uri)
                 else
                 {
                     // DataLake Uris have two allowed subdomains
-                    AccountName = GetAccountNameFromDomain(uri, Constants.DataLake.BlobUriSuffix) ??
-                        GetAccountNameFromDomain(uri, Constants.DataLake.DfsUriSuffix) ??
+                    AccountName = UriExtensions.GetAccountNameFromDomain(uri, Constants.DataLake.BlobUriSuffix) ??
```

Comment:
Can you use instance method style since these are extension methods?

Reply:
fixed.